config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -308,42 +340,16 @@ postsubmits:
           value: "us-west-2"
         securityContext:
           privileged: true
+        resources:
+          requests:
+            cpu: 1
+            memory: "4Gi"
```

Comment:
Adds the same requests that we are on the periodics to try and help with consistency of runs.